Mage.Client/src/main/java/mage/client/dialog/TableWaitingDialog.java

```diff
@@ -177,12 +180,11 @@ public void closeDialog() {
         if (updateTask != null) {
             updateTask.cancel(true);
         }
+
         this.chatPanel.disconnect();
         MageFrame.getUI().removeButton(MageComponents.TABLE_WAITING_START_BUTTON);
         this.removeDialog();
-        TableUtil.saveColumnWidthAndOrderToPrefs(tableSeats, KEY_TABLE_WAITING_COLUMNS_WIDTH, KEY_TABLE_WAITING_COLUMNS_ORDER);
-        PreferencesDialog.saveValue(PreferencesDialog.KEY_TABLE_WAITING_WIDTH, Integer.toString(getWidth()));
```

Comment:
Despite removing these offending lines the pre-tournament lobby still doesn't render the saved column width and position - despite successfully saving and restoring those values using `saveColumnWidthAndOrderToPrefs` and `setColumnWidthAndOrderFromPrefs`